tests.py

```diff
@@ -1,11 +1,61 @@
 import data
 import unittest
+import sys
 
 
-class DataTest(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.data = data.DATA
+class BackportTestCase(unittest.TestCase):
```

Comment:
just create these methods on the module level, then do something like:
``` python
if sys.version_info < (2, 7):
    unittest.TestCase.assertIn = assertIn
    unittest.TestCase.assertIsInstance = assertIsInstance
``` 
etc...